ui/src/components/songsView/SongsViewContainer.tsx

```diff
@@ -103,14 +112,32 @@ const SongsViewContainer: FC = (): ReactElement => {
             </Box>
           )}
           {!isMobile && (
-            <>
-              <Box sx={{ marginBottom: ['10px', '15vh'], width: '70%' }}>
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'row',
+                '@media (max-width:1340px)': {
+                  flexDirection: 'column',
+                },
+              }}
+            >
+              <Box
+                sx={{
+                  '@media (max-width:1340px)': { width: '100%', marginBottom: '20px' },
+                  '@media (min-width:1340px)': { width: '70%', marginBottom: '20px' },
+                }}
+              >
                 <SongsButtonCard song={song} />
               </Box>
-              <Box sx={{ width: '30%' }}>
+              <Box
+                sx={{
+                  '@media (max-width:1340px)': { width: '50%', marginBottom: '20px' },
+                  '@media (min-width:1340px)': { width: '30%' },
+                }}
```

Comment:
[minor]: should define this media min max width in a constant properly like u did above, else its gonna be super messy with so many magic numbers everywhere